server/conf/cms/secrets.sample.py

```diff
@@ -3,33 +3,7 @@
 ########################
 
 SECRET_KEY = 'replacethiswithareallysecureandcomplexsecretkeystring'
-DEBUG = False
-
-ALLOWED_HOSTS = ['0.0.0.0', '127.0.0.1', 'localhost', '*']
-
-LDAP_ENABLED = False
```

Comment:
Is `LDAP_ENABLED` okay to be true ([default](https://github.com/TACC/Core-CMS/blob/c7ff85a/taccsite_cms/_settings/auth.py#L19)) (e.g. security issue during local dev, is LDAP auth functional on CMS within local Portal)?

P.S. If it should be false, see [why setting `False` alone is ineffectual](https://github.com/TACC/Core-CMS/blob/c7ff85a/taccsite_cms/_settings/auth.py#L8-L19).